src/Components/Orders/index.js

```diff
@@ -281,10 +314,20 @@ export default class Orders extends Component {
             <Loader active={loading}>Loading orders...</Loader>
         </Dimmer>
 
-
+        <Input
+          action
+          type="search"
+          placeholder="Search..."
+          className="search-input search-padding"
+          onChange={this.handleSearchInput}>
+          <input />
+          <Select compact options={this.searchOptions} value={searchScope} onChange={this.handleSearchScope} />
```

Comment:
This was also a UI concession that I made due to limited time.  In a more robust search API, we could use inline tokens to specify query type kind of like the github search API.  e.g. email:foo@bar.com items:batwing